### PR TITLE
Remove references to obsolete logs directory

### PR DESCRIPTION
**Pull request by monika-o**
This pull request addresses #274 .

### CODE REVIEW

**Comment by monika-o** [comment]
I have replaced the logfiles in the scripts' headers by those defined in the respective rules in the snakefile.

**Review by MaGering — CHANGES_REQUESTED** [review]
Thanks for these changes! I have some minor changes / questions.

File: scripts/prepare_heat_demand.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 out_path2 : str
     ``results/_resources/ts_load_heat.csv``: path of output file with timeseries data as .csv
 logfile : str
-    ``logs/{scenario}.log``: path to logfile
+    ``results/_resources/load_heat.log``: path to logfile
```
[the remark this answers is not in the document]
Comment (monika-o) [reply scripts/prepare_heat_demand.py:23]: Thank you for your thought! From what I see in the snakefile, the script has two outputs, one being results/_resources/ts_load_heat.csv and the other one results/_resources/scal_load_heat.csv. It therefore seems logical to me that the log-file has a more general name.

[the remark this answers is not in the document]
Comment (MaGering) [reply scripts/prepare_heat_demand.py:23]: Makes sense. Thanks for looking it up!

File: scripts/table_costs_efficiencies.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
     ``results/_tables/technical_and_cost_assumptions_{scenario_key}.csv``: target path for
     the table.
 logfile : str
-    ``logs/{scenario}.log``: path to logfile
+    ``results/_tables/technical_and_cost_assumptions_{scenario_key}.log``: path to logfile
```
[the remark this answers is not in the document]
Comment (monika-o) [reply scripts/table_costs_efficiencies.py]: I agree that using {scenario} everywhere would be more consistent. I had used {scenario_key} in the docstring because that was the output name in the Snakefile. I've now changed both (see [a833d84](https://github.com/rl-institut/oemof-B3/pull/279/commits/a833d849c5be23f578057f8a887a96e27d78c35d)). Perhaps we could also change that in the cost-efficiencies-table in the input data?

[the remark this answers is not in the document]
Comment (MaGering) [reply scripts/table_costs_efficiencies.py]: Thank you for your changes. Tested it and it works.

Regarding `scenario_key` in the raw data, you have picked up a good point. I took a look and saw that we don't have any guidance yet regarding scenario setup in the docs. I think for using `oemof-B3` it would be helpful to describe in a chapter "scenario requirements" or "scenario setup" how the scenario `yml` file is structured and how it is linked to the raw data. Questions that arise: What distinction in the scenario keys is minimally necessary for oemof-B3 to run? Which `scenario_key` belongs to which raw data file? So far there is only one sentence (deprecated though) at the bottom of [this paragraph](https://oemof-b3.readthedocs.io/en/latest/model_structure.html#oemof-b3-resources). I would open up a new issue with this, what do you think @monika-o ?

[the remark this answers is not in the document]
Comment (monika-o) [reply scripts/table_costs_efficiencies.py]: That's a very good idea, I totally agree with you! It would be great if we had a chapter about the scenario setup.

I also agree that we should move that to a different pull request. Could you merge this one now @MaGering ?

File: Snakefile [hunk 3]
```diff
@@ -58,10 +58,10 @@ rule clean:
 # Rules for intermediate steps
 rule table_costs_efficiencies:
     input: "raw/scalars/costs_efficiencies.csv"
-    output: "results/_tables/technical_and_cost_assumptions_{scenario_key}.csv"
+    output: "results/_tables/technical_and_cost_assumptions_{scenario}.csv"
```
Comment (jnnr) [inline Snakefile:61]: `scenario_key` was intended because raw data is labeled by `scenario_key`s. Scenarios are formed by combining several scenario_keys. Can you revert these changes?

**Review by jnnr — CHANGES_REQUESTED** [review]
Looks good, thank you. Please consider my comment on the changes to the Snakefile

File: Snakefile (hunk 3 above)
Comment (monika-o) [reply Snakefile]: Ok, good to know. I have reverted that commit.

**Review by jnnr — APPROVED** [review]
